src/containers/studies/StudiesSagas.js

```diff
@@ -269,17 +266,32 @@ function* deleteStudyParticipantWorker(action :SequenceAction) :Generator<*, *,
 
     const { studyId, participantEntityKeyId, participantId } = action.value;
 
-    const { response, timeout } = yield race({
-      response: call(ChronicleApi.deleteStudyParticipant, participantId, studyId),
-      timeout: delay(1000 * 10) // 10 seconds
-    });
-    if (response && response.error) throw response.error;
+    yield call(ChronicleApi.deleteStudyParticipant, participantId, studyId);
```

Comment:
You still need to check the response...

```
const response = yield call(ChronicleApi.deleteStudyParticipant, participantId, studyId);
if (response.error) throw response.error;
```

Reply:
I don't think we need to explicitly check for errors since they will still be caught in the `catch` block

Reply:
Right! This is not `lattice-sagas`.